Homework7/FightLogic.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Homework7
+{
+    class FightLogic
+    {
+        public static Unit GenerateFighter()
```

Comment:
I think, index should be optional parameter here and generate only in case when user didn't send it.

Reply:
fixed